thirdparty/swan/SwanReadADCGrid.ftn90

```diff
@@ -197,7 +197,10 @@ subroutine SwanReadADCGrid
 !NADC             vmark(ivert ) = vm
 !NADC             vmark(ivert1) = vm
 !ADC            vmark(find(node_dict,ivert )) = vm
-!ADC            vmark(find(node_dict,ivert1)) = vm
+!ADC            n3 = find(node_dict,ivert1)
```

Comment:
Tagging @caseydietrich for awareness as he works on his implementation of a new version of SWAN